reference/webhooks.v3.yml

```diff
@@ -1,463 +1,644 @@
-              Example:
-                value:
-                  scope: '{$$.env.scope}'
-                  destination: '{$$.env.destination}'
-                  is_active: true
-        x-examples:
-          application/json:
-            scope: store/order/*
-            destination: 'https://665b65a6.ngrok.io/webhooks'
-            headers: {}
-      tags:
-        - Webhooks
-    get:
+              $ref: '#/components/schemas/WebhookCreate'
+        required: true
       responses:
         '200':
-          $ref: '#/components/responses/webhooks_Resp'
-      summary: Get Webhooks
-      description: |-
-        Returns a list of all webhooks on a store associated to the `client_id` used to authenticate the request.
-
-        *Note: BigCommerce determines the `client_id` from the `access_token`.*
-      operationId: getAllWebhooks
+          description: Webhook created.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WebhookResponse'
+        '400':
+          description: Bad request.
+        '401':
+          description: Unauthorized - the v3 Auth client ID or token in the request are not a valid combination for this store.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '422':
+          description: Invalid JSON request body - missing or invalid data.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
       tags:
         - Webhooks
       parameters:
         - $ref: '#/components/parameters/Accept'
         - $ref: '#/components/parameters/Content-Type'
-  '/hooks/{id}':
+    parameters: []
+  '/hooks/{webhook_id}':
     get:
-      responses:
-        '200':
-          $ref: '#/components/responses/webhook_Resp'
-      description: Return a webhook by ID.
       operationId: getWebhook
       summary: Get a Webhook
-      tags:
-        - Webhooks
+      description: |
+        Returns a webhook by ID.
+
+
+        <!-- type:info -->
+
+        <div class="HubBlock--callout">
+        <div class="CalloutBlock--info">
+        <div class="HubBlock-content">
+
+        > ### Note
+        > * Webhooks endpoints are available on both the V2 and V3 REST API.
+
+        </div>
+        </div>
+        </div>
       parameters:
+        - $ref: '#/components/parameters/WebhookId'
         - $ref: '#/components/parameters/Accept'
         - $ref: '#/components/parameters/Content-Type'
-    parameters:
-      - name: id
-        in: path
-        required: true
-        schema:
-          type: string
-    put:
       responses:
         '200':
-          $ref: '#/components/responses/webhook_Resp'
+          description: Successful operation.
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/WebhookResponse'

```

Comment:
This schema misses such fields as `client_id`, `created_at`, `updated_at`, `store_hash`. It should be the same webhook schema as in `getAllWebhooks`